Coriolis/Coriolis.html

```diff
@@ -95,35 +95,35 @@ <h3 data-i18n="attributes">Grundegenskaper</h3>
                             <h4 data-i18n="strength">Styrka</h4>
                         </td>
                         <td class="sheet-value">
-                            <button type='roll' value='&{template:coriolis} {{name=Styrka}} {{roll=[[@{sty}d6>6]]}}' name='roll_str'></button>
-                            <input type="number" name="attr_sty" value="0">
+                            <button type='roll' value='&{template:coriolis} {{name=^{strength}}} {{roll=[[@{strength}d6>6]]}}' name='roll_strength'></button>
+                            <input type="number" name="attr_strength" value="0">
```

Comment:
You are changing the names of a bunch of attributes in the commit. This will end up erasing a bunch of values for anyone that is already using the sheet since the reference to the value they have is changing out from under them. We want to avoid this.

You have 2 options:
- Leave the attributes in Swedish.
- Add a trigger in the sheetworkers that will update all of the values from the old attr to the new one.